website/src/components/CalcSpectrum.tsx

```diff
@@ -291,6 +294,7 @@ export const CalcSpectrum: React.FC = () => {
 
             <Grid item xs={12}>
               <Species
+                isEquiub={isNonEquilibrium}
```

Comment:
At least to me, this parameter name reads "is equilibrium". Would you change the parameter name to `isNonEquilibrium`?